Sources/TokamakCore/Views/List.swift

```diff
@@ -0,0 +1,261 @@
+                                    @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+    where Content == ForEach<Data, ID, HStack<RowContent>>,
+    Data: RandomAccessCollection, ID: Hashable, RowContent: View {
+    self.init(selection: selection) {
+      ForEach(data, id: id) { row in
+        HStack {
+          rowContent(row)
+        }
+      }
+    }
+  }
+
+  public init<Data, RowContent>(_ data: Data,
+                                selection: Binding<SelectionValue?>?,
+                                @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+    where Content == ForEach<Data, Data.Element.ID, HStack<RowContent>>,
+    Data: RandomAccessCollection, RowContent: View, Data.Element: Identifiable {
+    self.init(selection: selection) {
+      ForEach(data) { row in
+        HStack {
+          rowContent(row)
+        }
+      }
+    }
+  }
+
+  // - MARK: Range initializers
+  public init<RowContent>(_ data: Range<Int>,
+                          selection: Binding<Set<SelectionValue>>?,
+                          @ViewBuilder rowContent: @escaping (Int) -> RowContent)
+    where Content == ForEach<Range<Int>, Int, HStack<RowContent>>, RowContent: View {
+    self.init(selection: selection) {
+      ForEach(data) { row in
+        HStack {
+          rowContent(row)
+        }
+      }
+    }
+  }
+
+  public init<RowContent>(_ data: Range<Int>,
+                          selection: Binding<SelectionValue?>?,
+                          @ViewBuilder rowContent: @escaping (Int) -> RowContent)
+    where Content == ForEach<Range<Int>, Int, HStack<RowContent>>, RowContent: View {
+    self.init(selection: selection) {
+      ForEach(data) { row in
+        HStack {
+          rowContent(row)
+        }
+      }
+    }
+  }
+
+  // - MARK: OutlineGroup initializers
+
+  // FIXME: Implement OutlineGroup
+//  public init<Data, RowContent>(_ data: Data,
+//                                children: KeyPath<Data.Element, Data?>,
+//                                selection: Binding<Set<SelectionValue>>?,
+//                                @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+//  where Content == OutlineGroup<Data,
+//                                Data.Element.ID,
+//                                HStack<RowContent>,
+//                                HStack<RowContent>,
+//                                DisclosureGroup<HStack<RowContent>, OutlineSubgroupChildren>>,
+//        Data : RandomAccessCollection,
+//        RowContent : View,
+//        Data.Element : Identifiable
+
+//  public init<Data, ID, RowContent>(_ data: Data,
+//                                    id: KeyPath<Data.Element, ID>,
+//                                    children: KeyPath<Data.Element, Data?>,
+//                                    selection: Binding<Set<SelectionValue>>?,
+//                                    @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/List.swift;
  Line: 165;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 102 characters (`line_length`)